tests/js/spec/views/settings/organizationDeveloperSettings/subscriptionBox.spec.jsx

```diff
@@ -34,4 +36,39 @@ describe('SubscriptionBox', () => {
     wrapper.setProps({disabled: true});
     expect(wrapper.find('Tooltip').prop('disabled')).toBe(false);
   });
+
+  describe('error.created resource subscription', () => {
+    beforeEach(() => {
+      onChange = jest.fn();
+      wrapper = mount(
+        <SubscriptionBox
+          resource="error"
+          checked={false}
+          disabled={false}
+          onChange={onChange}
+          organization={org}
+        />,
+        TestStubs.routerContext()
+      );
+    });
+
+    it('checkbox disabled without integrations-event-hooks flag', () => {
+      expect(wrapper.find('Tooltip').prop('disabled')).toBe(false);
```

Comment:
I don't think this assertion matches the test name. It's asserting it's _not_ disabled.